Plan details UI changes for MTV 2.6.3 (#528)
kubev2v/forklift-documentation: documentation/modules/creating-migration-quick-plan-vm.adoc

. Click *Add mapping* to edit a suggested network mapping or a storage mapping, or to add one or more additional mappings.
. Click *Create migration plan*.
+

> {project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. The details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan.

Reviewer: ```suggestion
{project-short} validates the migration plan, and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. The details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} validates the plan again.
```

. VMware source providers only (All optional):

> * *Preserving the status ID of Windows VMs*: By default, Virtual NICs (VNICs) change during the migration process, which results in Windows VNICs that are set with static IP in vSphere to lose their IP. To avoid this, click the Edit icon next to *Preserve static IPs*, toggle the *Whether to preserve the static IPs* switch in the window that opens, and then click *Save*.

Reviewer: ```suggestion
* *Preserving the status ID of Windows VMs*: By default, Virtual NICs (VNICs) change during the migration process. This results in Windows VNICs, which are set with a static IP in vSphere, losing their IP. To avoid this, click the Edit icon next to *Preserve static IPs* and toggle the *Whether to preserve the static IPs* switch in the window that opens. Then click *Save*.
```
* Try to use shorter sentences, particularly when it is procedure.
* It is either `set with a static IP...` or `set with static IPs`

* *Entering a list of decryption passphrases for disks encrypted using Linux Unified Key Setup (LUKS)*: To enter a list of decryption passphrases LUKS-encrypted devices, in the *Settings* section, click the Edit icon next to *Disk decryption passphrases*, enter the passphrases, and then click *Save*. You do not need to enter the passphrases in a specific order -- for each LUKS-encrypted device, {project-short} tries each passphrase until one unlocks the device.

> 

Reviewer: See earlier comments

+
{project-short} uses the following format for disk location: `/dev/sdX#`, where X represents the disk number in letters (that is the first disk would be represented as 'a', the second disk as 'b', and so on) and # represents the partition that holds the operating system inside the disk (starting from 1). For example, if the second disk is the root device and the operating system is on the disk's second partition, the format would be: `/dev/sdb2`. After you enter the boot device, click *Save*.
+
If the conversion fails because the boot device provided is incorrect, it is possible to get the correct information by looking at the conversion pod logs.

. {rhv-short} source providers only (Optional):

* *Preserving the CPU model of VMs that are migrated from {rhv-short}*: Generally, the CPU model (type) for {rhv-short} VMs is set at the cluster level, but it can be set at the VM level, which is called a custom CPU model.
By default, {project-short} sets the CPU model on the destination cluster as follows: {project-short} preserves custom CPU settings, but sets the CPU model for VMs whose CPU model is set at the cluster level.
+

> To preserve the cluster-level CPU model of your {rhv-short} VMs, in the *Settings* section, click the Edit icon next to *Preserve CPU model*. toggle the *Whether to preserve the CPU model* switch, and then click *Save*.

Reviewer: ```suggestion
To preserve the cluster-level CPU model of your {rhv-short} VMs, in the *Settings* section, click the Edit icon next to *Preserve CPU model*. Toggle the *Whether to preserve the CPU model* switch, and then click *Save*.
```

* basic grammar

> 

Reviewer 2: shall we extract that section to another file and import it in both places?

Author: Ideally, I'd have a snippet for of these 4 additions. But so far, I have not been able to use snippets inside numbered lists without messing up the numbering.  What might work better is to create one module and use conditional text for the beginning steps. Let's get the text set first and then, I'll talk with Andy and Radhika.

Reviewer: @RichardHoch - If you want to send me the snippet and the line number you would like the snippet included i can have a look later today

Do you remember when we forced the numbering last time, by using `[start]` ? We could try something similar here

Author: @anarnold97 That sounds good. I have another idea I'll run by you offline. Depends on what Asciidocs will let us do.

. Click *Add mapping* to edit a suggested network mapping or a storage mapping, or to add one or more additional mappings.
. Click *Create migration plan*.
+
{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. The details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan.

. VMware source providers only (All optional):

* *Preserving the CPU type of VMs that are migrated from {rhv-short}*: By default, {project-short} sets the CPU type (CPU model and any flags) of migrated VMs if the type is not set specifically for the VM. This can be an issue for migrations from {rhv-short} because the CPU type of {rhv-short} VMs can be set at the cluster level, in which case {project-short} might change the type.
+

> To preserve the CPU type of your {rhv-short} VMs, in the *Settings* section, click the Edit icon next to *Preserve CPU model*. toggle the *Whether to preserve the CPU model* switch, and then click *Save*.

Reviewer 3: same as above

Author: Fixed.
